style.css

```diff
@@ -8,23 +8,54 @@
 body {
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  justify-content: flex-start;
   align-items: center;
   align-content: center;
-  gap: 20px;
+  gap: 30px;
   margin: 20px;
+  height: 100vh;
 }
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md) for this project, " `Make sure that when you click on any of the links, the main area changes to display only one of the three content sections. (Hint: you can simply add and remove CSS classes to display or hide the content.)
To change the contents of the page, don't use any library or framework, use only plain JavaScript and CSS.` ", please make sure that when any of the links are clicked, the main area changes to display only one of the three content sections.